apps/channels/views.py

```diff
@@ -32,6 +32,13 @@ def new_twilio_message(request):
     return HttpResponse()
 
 
+@csrf_exempt
```

Comment:
limit to POST requests:

```suggestion
@csrf_exempt
@require_POST
```

Reply:
You still need to add this (you added it to the twilio endpoint but not to this one)